resources/metadata/sonar.yaml

```diff
@@ -41,7 +41,7 @@ spec:
           - STAGES
           - STEPS
       - name: customTlsCertificateLinks
-        type: string
+        type: "[]string"
         description: List of comma-separated download links to custom TLS certificates. This is required to ensure trusted connections to instances with custom certificates.
```

Comment:
Description should be fixed, at least no mention of "comma-separated", no?

Reply:
```suggestion
        description: List of download links to custom TLS certificates. This is required to ensure trusted connections to instances with custom certificates.
```